css/reset.css

```diff
@@ -0,0 +1,46 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: var(--white-background);
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+ul {
+  list-style-type: none;
+}
+
+input {
+  outline: none;
```

Comment:
Using outline none is not considered good practice, instead of that use
 outline: 2px solid transparent; outline-offset: 2px; 
suggested by Arpan

Reply:
fixed!